zebra-scanner/Cargo.toml

```
@@ -0,0 +1,44 @@
+[package]
+name = "zebra-scanner"
+version = "1.0.0-beta.30"
```

Team Decision: Should this unaudited and untested crate have an alpha version instead? Should it have a prerelease version?

```suggestion
version = "0.1.0-alpha.0"
```

---

> Team Decision: Should this unaudited and untested crate have an alpha version instead? Should it have a prerelease version?

An alpha version would be good, I don't think it currently needs a prerelease version.

---

I feel like it's not really `1.0.0` quality yet, and we expect that the API might change. So that's why I suggested `0.1.0`.

But for simplicity we could use `1.0.0`?

---

> I feel like it's not really `1.0.0` quality yet, and we expect that the API might change. So that's why I suggested `0.1.0`.
> 
> But for simplicity we could use `1.0.0`?

Oh. I may have misunderstood, I do have an opinion on that, `0.1.0` feels much more consistent with the versioning schemes of other crates.

---

I think that was my phrasing, `prerelease` can mean `alpha` or "lower than `1.0.0`"